templates/chatbots/partials/channel_dialog.html

```diff
@@ -0,0 +1,47 @@
+{% load form_tags %}
+<dialog id="{% if channel %}channel_edit_modal_{{ channel.id }}{% else %}channel_create_modal{% endif %}" class="modal" open>
+  <div class="modal-box">
+    <h3 class="font-bold text-lg">
+      {% if channel %}
+        Edit {{ channel.platform_enum.label }} Channel
+      {% else %}
+        Link with {{ platform.label }}
+      {% endif %}
+    </h3>
+    <form method="post"
+      {% if channel %}
+        action="{% url "channels:channel_edit_dialog" request.team.slug experiment.id channel.id %}?origin={{ request.GET.origin|default:'chatbots' }}"
+      {% else %}
+        action="{% url "channels:channel_create_dialog" request.team.slug experiment.id platform.value %}?origin={{ request.GET.origin|default:'chatbots' }}"
+      {% endif %}
+      x-data="{ buttonDisabled: false }"
+      x-on:submit.debounce="buttonDisabled = true">
```

Comment:
if we use htmx for the form submission we can remove these and instead use: `hx-disabled-elt="input, select, button[type='submit']"`